front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/OperationalPointPopupDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { type SetStateAction } from 'react';
+
+import { Select } from '@osrd-project/ui-core';
+import { useTranslation } from 'react-i18next';
+
+import type { PathStep } from 'reducers/osrdconf/types';
+
+import type { FeatureInfoClick } from '../types';
+
+type OperationalPointPopupDetailsProps = {
+  operationalPoint: FeatureInfoClick;
+  clickedOp: Extract<PathStep, { uic: number }> & {
+    tracks: {
+      trackName?: string;
+      coordinates?: number[];
+    }[];
+  };
+  selectedTrack: {
+    trackName?: string;
+    coordinates?: number[];
+  };
+  setSelectedTrack: React.Dispatch<
+    SetStateAction<{ trackName?: string; coordinates?: number[] } | undefined>
+  >;
+};
+
+const OperationalPointPopupDetails: React.FC<OperationalPointPopupDetailsProps> = ({
```

Comment:
Don't use React.FC please :pray: 
```suggestion
const OperationalPointPopupDetails = ({
```

Reply:
Done